frontend/admin/src/js/components/Table/Table.tsx

```diff
@@ -170,6 +177,27 @@ function Table<T extends Record<string, unknown>>({
   const intl = useIntl();
   const methods = useForm();
 
+  React.useEffect(() => {
+    setSearchParams((previous) => {
+      previous.set("pageSize", `${pageSize}`);
+      previous.set("pageIndex", `${pageIndex}`);
+      if (sortBy) {
+        const newSortBy = encodeURIComponent(JSON.stringify(sortBy));
+        if (previous.get("sortBy") !== newSortBy) {
+          previous.set("sortBy", newSortBy);
+        }
+      }
+      if (hiddenColumns) {
+        const newHiddenColumns = hiddenColumns.join(",");
```

Comment:
Should hidden columns also be wrapped in encodeURIComponent? I would have done so in my commit, but I wasn't sure if there was a reason you had done so for sortBy but not hiddenColumns.

Reply:
They don't need to. The join is effectively accomplishing the same task. It was used with `sortBy` since it is an object and a little trickier to convert nicely.